lib/jekyll/static_file.rb

```diff
@@ -144,7 +144,11 @@ def copy_file(dest_path)
       if @site.safe || Jekyll.env == "production"
         FileUtils.cp(path, dest_path)
       else
-        FileUtils.copy_entry(path, dest_path)
+        begin
+          FileUtils.ln(path, dest_path)
+        rescue
```

Comment:
I don't think Windows is the issue. It appears to define [`HAVE_LINK`](https://github.com/ruby/ruby/blob/trunk/win32/Makefile.sub#L730).

The real issue is that `link` failing need not be a user error. It could simply be that `_site` points to a different file system.